### DISCUSSION TITLE
How to add six wheels car?

### DISCUSSION THREAD

**Discussion by HertzDav** [carla-simulator/carla#5237] [Q&A] [answered]
Hello everyone！
I would like to ask how to join the six-wheeled car, I hope you can help me, thank you!

**Comment by jackbart94** [comment, marked as answer]
Hi - it's only available in the current ``dev`` branch. To do so, you need to import a vehicle as you would do for a 4 wheeled one, but use the BaseVehiclePawnNW instead, and select which wheels are affected by the engine. It's not that straightforward, I will make a tutorial to expand the current documentation.

**Reply by HertzDav** [reply]
Thanks！！！

**Reply by HertzDav** [reply]
Excuse me, where can I see your tutorial?

**Reply by CharlesXu1124** [reply]
same question, where can I see your tutorial? It would be useful for me as well to add a multi-wheeled vehicle other than 4!

**Reply by Watson52** [reply]
Hi everyone! I want to know could I import a dual-wheel robot like the Segway-Ninebot?

**Reply by noobC7** [reply]
Hope to see your turtorial!

**Reply by jackbart94** [reply]
It should be in the documentation pretty soon :) @MattRowe18 will tell you guys when

**Reply by sojuzzoa** [reply]
Any update on this?

**Reply by FrankZTH** [reply]
Already add into document at April.
